src/ProjectFileTools/extension.vsixmanifest

```diff
@@ -13,6 +13,7 @@
   <Assets>
     <Asset Type="Microsoft.VisualStudio.VsPackage" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.NuGetSearch|" />
+    <Asset Type="Microsoft.VisualStudio.MefComponent" Path="|ProjectFileTools.MSBuild|" />
```

Comment:
I think we're missing the line that added the Xml assembly as an Assembly type asset